core/src/main/java/org/hisp/dhis/android/core/arch/api/internal/HttpServiceClient.kt

```diff
@@ -44,11 +44,11 @@ import okhttp3.ResponseBody.Companion.toResponseBody
 import org.koin.core.annotation.Singleton
 
 @Singleton
-internal class KtorServiceClient(
-    private val client: HttpClient,
+class HttpServiceClient(
+    val client: HttpClient,
     var baseUrl: String = "https://temporary-dhis-url.org/api/",
 ) {
-    internal suspend inline fun <reified T> request(
+    suspend inline fun <reified T> request(
```

Comment:
If this method is public, the parameter `requestMethod` is exposed, which exposes a ktor class `HttpMethod`. Couldn't it be hidden? Maybe the annotation PublishedApi can help here